examples/client/Locomotion/src/Components/CardDetails/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+import { useRoute } from '@react-navigation/native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import { View } from 'react-native';
+import ConfirmationPopup from '../../popups/ConfirmationPopup';
+import { getLastFourForamttedShortLong } from '../../pages/Payments/cardDetailUtils';
+import { MAIN_ROUTES } from '../../pages/routes';
+import { Card } from '../../pages/Account';
+import {
+  CardsContainer,
+  CardsTitle,
+  Container,
+  LogoutContainer,
+} from '../../pages/Account/styled';
+import i18n from '../../I18n';
+import PageHeader from '../PageHeader';
+import { PaymentMethodInterface } from '../../context/payments/interface';
+import { getTogglePopupsState } from '../../context/state';
+import PaymentsContext from '../../context/payments';
+import deleteIcon from '../../assets/delete.svg';
+import { DeleteIcon, DeleteText } from './styled';
+
+
+export default ({
```

Comment:
create variable and then export it
const CardDetails = ...

export default CardDetails


Also, add interface